internal/cli/cmd/playground/destroy.go

```diff
@@ -222,12 +227,12 @@ func (o *destroyOptions) deleteClustersAndUninstallKB() error {
 // delete all clusters created by KubeBlocks
 func (o *destroyOptions) deleteClusters(dynamic dynamic.Interface) error {
 	var err error
-	ctx := context.Background()
-
+	ctx, cancel := context.WithTimeout(context.Background(), o.apiTimeout)
```

Comment:
Does this timeout apply to a single API call or all API calls? If the timeout for all API calls is 5 seconds, is this reasonable?

Reply:
> Does this timeout apply to a single API call or all API calls? If the timeout for all API calls is 5 seconds, is this reasonable?

Now, API timeouts are only applied in the 'destroy' operation within the playground.